css/styles.css

```diff
@@ -0,0 +1,52 @@
+body 	{
+	font-family: cursive;
+	font-size: 18px;
+	margin-top: 0px;
+	margin-right: 0px;
+	background-color: #FCDEBC;
+}
+
+nav	{
+	font-family: serif;
+	font-size:  16px;
+	text-align: right;
+	padding: 5px 15px;
+}
+
+img 	{
+	width: 50%;
```

Comment:
@gerriecho Nice job with the percentage width as opposed to a fixed pixel width.